packages/compass-collection/src/components/workspace/workspace.jsx

```diff
@@ -245,24 +225,26 @@ class Workspace extends PureComponent {
    * @returns {Component} The rendered component.
    */
   render() {
+    const selectedTabIndex = this.props.tabs.findIndex((tab) => tab.isActive);
+
     return (
       <div className={styles.workspace}>
-        <div className={styles['workspace-tabs']}>
-          <div onClick={this.props.prevTab} className={styles['workspace-tabs-prev']}>
-            <i className="fa fa-chevron-left" aria-hidden/>
-          </div>
-          <div className={styles['workspace-tabs-container']}>
-            {this.renderTabs()}
-            <CreateTab
-              createNewTab={this.onCreateNewTab}
-            />
-          </div>
-          <div className={styles['workspace-tabs-right']}>
-            <div onClick={this.props.nextTab} className={styles['workspace-tabs-next']}>
-              <i className="fa fa-chevron-right" aria-hidden/>
-            </div>
-          </div>
-        </div>
+        <WorkspaceTabs
+          onCreateNewTab={this.onCreateNewTab}
+          onMoveTab={this.props.moveTab}
+          onSelectTab={this.props.selectTab}
+          onCloseTab={this.props.closeTab}
+          tabs={this.props.tabs.map(tab => ({
+            title: tab.activeSubTabName,
+            subtitle: tab.namespace,
+            tabContentId: tab.id,
+            renderIcon: (iconProps) => renderTabIcon(
```

Comment:
Why can't this be just `icon: <Elem>`?

Reply:
It could, where would the styling be applied then? I'm cool with whatever here.

Reply:
ah cause we need to apply some style to the icon, makes sense